test/fw/ptl/utils/plugins/ptl_test_info.py

```diff
@@ -40,11 +40,31 @@
 from nose.plugins.base import Plugin
 from ptl.utils.pbs_testsuite import PBSTestSuite
 from ptl.utils.plugins.ptl_test_tags import TAGKEY
+from ptl.utils.pbs_testsuite import REQUIREMENTS_KEY
+from ptl.utils.pbs_testsuite import default_requirements
 from copy import deepcopy
 
 log = logging.getLogger('nose.plugins.PTLTestInfo')
 
 
+def get_eff_requirements(ts_requirements=None, tc_requirements=None):
```

Comment:
Optional comment: rename fn name to `get_effective_reqs` and rename `ts_requirements` to `ts_reqs` and `tc_requirements` to `tc_reqs`